src/app/components/Text/index.jsx

```diff
@@ -13,13 +13,23 @@ const StyledParagraph = styled.p`
     line-height: 1.375rem;
   }
 `;
+const StyledItalic = ({ children }) => <i>{children}</i>;
+const StyledBold = ({ children }) => <b>{children}</b>;
 
 const Text = ({ text, options }) => {
   if (!text) return null;
 
   return <Markdown options={options}>{text}</Markdown>;
 };
 
+StyledItalic.propTypes = {
+  children: string.isRequired,
```

Comment:
`children` should be `node` props, not `string`. https://reactjs.org/docs/typechecking-with-proptypes.html